es6-const-let/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    var greetings = [
+      'hello!',
+      'konichiwa!',
+      'nihao',
+      'bonjour',
+      'gutten tag',
+      'ola'
+    ];
```

Comment:
Please double-check that you've converted _all_ variables to `const` or `let`, as appropriate:

---

![image](https://user-images.githubusercontent.com/16143771/127083363-2b7aedc2-9601-45db-8b2b-1d3343cccd29.png)

Reply:
edited